### PR TITLE
Converting CreateDuplicateUrlCmsPageEntityTest to MFTF #616

### PR DESCRIPTION
**Pull request by DenisSaltanahmedov**
### Description (*)
Convert CreateDuplicateUrlCmsPageEntityTest to MFTF

### Fixed Issues (if relevant)
Variations to be converted:
https://github.com/magento/magento-functional-tests-migration/tree/2.3-develop/dev/tests/functional/tests/app/Magento/Cms/Test/TestCase/CreateDuplicateUrlCmsPageEntityTest.xml#L10

1. magento/magento-functional-tests-migration#616: Convert CreateDuplicateUrlCmsPageEntityTest to MFTF

### Steps(*)
1. Login to admin
2. Go to create CMS Page
3. Fill the necessary fields
4. Save page
6. See error message
7. Logout

### Contribution checklist (*)
 - [*] Pull request has a meaningful description of its purpose
 - [*] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by DenisSaltanahmedov** [comment]
@melnikovi Hi, I have update my changes, could you check it, please.

File: app/code/Magento/Cms/Test/Mftf/Test/CreateDuplicateUrlCmsPageEntityTest.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <test name="CreateDuplicateUrlCmsPageEntityTest">
+        <annotations>
+            <features value="Cms"/>
+            <title value="Create Duplicate Url CMS Content Page"/>
+            <description value="Create Duplicate Url CMS Content Page"/>
+            <group value="cMSContent"/>
+            <group value="mtf_migrated"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginActionGroup" stepKey="loginGetFromGeneralFile"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Cms/Test/Mftf/Test/CreateDuplicateUrlCmsPageEntityTest.xml]:
```suggestion
            <actionGroup ref="AdminLoginActionGroup" stepKey="loginGetFromGeneralFile"/>
```

File: app/code/Magento/Cms/Test/Mftf/Test/CreateDuplicateUrlCmsPageEntityTest.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        </before>
+        <after>
+            <actionGroup ref="logout" stepKey="logout"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Cms/Test/Mftf/Test/CreateDuplicateUrlCmsPageEntityTest.xml]:
```suggestion
            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
```

File: app/code/Magento/Cms/Test/Mftf/ActionGroup/VerifySaveCMSPageFailureOnDuplicationUrlActionGroup.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="VerifySaveCMSPageFailureOnDuplicationUrlActionGroup">
```
Comment (lbajsarowicz) [inline app/code/Magento/Cms/Test/Mftf/ActionGroup/VerifySaveCMSPageFailureOnDuplicationUrlActionGroup.xml]:
```suggestion
    <actionGroup name="AssertAdminFailureOnSaveCMSPageWithDuplicatedUrlActionGroup">
```

File: app/code/Magento/Cms/Test/Mftf/Test/CreateDuplicateUrlCmsPageEntityTest.xml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        </after>
+        <actionGroup ref="CreateNewPageWithDuplicateUrlActionGroup" stepKey="CreateNewPageWithDuplicateUrlActionGroup">
+            <argument name="PageTitle" value="Page title"/>
+            <argument name="UrlKey" value="no-route"/>
+        </actionGroup>
+        <actionGroup ref="VerifySaveCMSPageFailureOnDuplicationUrlActionGroup" stepKey="FailureSavedCMSPageMsgOnGrid"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Cms/Test/Mftf/Test/CreateDuplicateUrlCmsPageEntityTest.xml]:
```suggestion
        <actionGroup ref="AssertAdminFailureOnSaveCMSPageWithDuplicatedUrlActionGroup" stepKey="assertFailureSavedCMSPageMsgOnGrid"/>
```

**Pull request merged**